catroweb.yaml

```diff
@@ -1121,3 +1216,99 @@ components:
           type: number
           example: 0.14381762458251943
           description: filesize in megabytes
+
+    NotificationsId:
+      type: integer
+      minimum: 0
+      description: Id of a notification
+      default: 0
+      example: 2
+
+    NotificationsType:
+      type: string
+      example: reaction
```

Comment:
Please be careful with naming conventions (i know it is in another section but still.).

Also type should be an enum which should default to `all`.